front/src/modules/simulationResult/components/ChartHelpers/ChartHelpers.ts

```diff
@@ -278,6 +316,24 @@ export const interpolateOnPosition = (
   return null;
 };
 
+// Interpolation of cursor based on space position
+// ['position', 'speed']
+export const interpolateOnPositionV2 = (
```

Comment:
I think interpolateOnPositionV2 is the same as interpolateOnPosition

Reply:
Good catch, done.